ops/lib.rs

```diff
@@ -360,7 +360,9 @@ fn codegen_fast_impl(
           if slices.get(&idx).is_some() {
             return quote! {
               match unsafe { &* #ident }.get_storage_if_aligned() {
-                Some(s) => s,
+                Some(s) => {
+                  s
+                },
```

Comment:
🤨

Reply:
some stuff left to do...blocked on rusty_v8 release